src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1495,11 +1488,10 @@ internal override void Check(IHostEnvironment env)
             _outputColumns = outCols.ToArray();
         }
 
-        internal SdcaBinaryTrainer(IHostEnvironment env, Arguments args,
-            string featureColumn, string labelColumn, string weightColumn = null)
-            : base(env, args, TrainerUtils.MakeBoolScalarLabel(labelColumn), TrainerUtils.MakeR4ScalarWeightColumn(weightColumn))
+        internal SdcaBinaryTrainer(IHostEnvironment env, Options options)
```

Comment:
Same comment here as I made with the SDCARegression -- does SdcaBinary not have common args or is it always options?

Reply:
we will eventually have just 1 constructor  (the one with `Options`)

#2100

---
In reply to: [248866446](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248866446) [](ancestors = 248866446)